metabrainz/templates/index/projects.html

```diff
@@ -0,0 +1,134 @@
+{% extends 'base.html' %}
+
+{% block title %}Projects - MetaBrainz Foundation{% endblock %}
+
+{% block content %}
+<h1 class="page-title">The MetaBrainz Foundation Projects</h1>
+<div id="projects">
+  
+  <div class="media">
+    <div class="media-left media-middle">
+      <a href="https://musicbrainz.org/">
+        <img class="media-object logo" src="{{ url_for('static', filename='img/projects/MBnt.png') }}" alt="MusicBrainz logo">
```

Comment:
What is the "nt" in the filenames? And is there are reason to not simple use the full project name rather than an abbreviation?